examples/configmaps/cluster-controller/kafka-ephemeral.yaml

```diff
@@ -14,9 +14,27 @@ data:
   zookeeper-image: "strimzi/zookeeper:latest"
   zookeeper-healthcheck-delay: "15"
   zookeeper-healthcheck-timeout: "5"
-  KAFKA_DEFAULT_REPLICATION_FACTOR: "3"
-  KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: "3"
-  KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: "3"
+  kafka-config: |-
+    {
+      "advertised.listeners": "PLAINTEXT://:1234",
```

Comment:
It's forbidden, do we want it here to show that the forbidden checking works fine ?